src/Sylius/Bundle/AdminBundle/templates/shared/grid/action/delete.html.twig

```diff
@@ -1,5 +1,12 @@
 {% set id = options.link.parameters['id']|default(data.id)  %}
 
-{% set path = options.link.url|default(path(options.link.route|default(grid.requestConfiguration.getRouteName('delete')), options.link.parameters|default({'id': id}))) %}
+{% if options.link.url is defined %}
```

Comment:
Pointing out just in case, since we have disabled state now,

```twig
{% if options.link.url is defined %}
```

will crash if `options` or `options.link` are undefined.

Personally, I would rather not support this case and receive the error as is if options are configured badly (If it's even possible with options validation in grid configuration).

Reply:
are you sure it will crash? I'm not sure at all.

Reply:
Could you check that to be sure? 🤔

Reply:
According to docs -- yes,

https://twig.symfony.com/doc/3.x/tests/defined.html
![image](https://github.com/user-attachments/assets/7cfc9a55-45b0-4e33-9d71-8daf5c08264e)

but in playground looks alright

https://twig.symfony.com/play?data=eyJ0ZW1wbGF0ZXMiOltbImluZGV4LnR3aWciLCJ7JSBzZXQgb3B0aW9ucyA9IHtcImxpbmtcIjoge1widXJsXCI6ICdleGFtcGxlLmNvbSd9fSAlfVxuXG57JSBpZiBvcHRpb25zLm5vdF9leGlzdHMudXJsIGlzIGRlZmluZWQgJX1cbnt7IG9wdGlvbnMubm90X2V4aXN0cy51cmwgfX1cbnslIGVuZGlmICAlfSJdXSwiY29udGV4dCI6eyJuYW1lIjoiV29ybGQifSwidmVyc2lvbiI6IjMuMTkuMCIsIm9wdGlvbnMiOnsic3RyaWN0X3ZhcmlhYmxlcyI6dHJ1ZSwiY2hhcnNldCI6IlVURi04IiwiYXV0b2VzY2FwZSI6IiJ9fQ%3D%3D